packages/node_modules/@ciscospark/widget-meet/src/container.js

```diff
@@ -55,10 +56,15 @@ export class MeetWidget extends Component {
       isActive,
       isIncoming,
       avatarImage,
-      displayName,
       localVideoPosition
     } = this.props;
     const {formatMessage} = intl;
+    const {avatarId} = this.props.spaceDetails;
+    let {displayName} = this.props;
+
+    if (!displayName) {
+      displayName = this.props.spaceDetails.title;
+    }
```

Comment:
Would this be more performant if it was in the selector instead of here in the render?

Reply:
Good call, will update this logic